setup.py

```diff
@@ -37,6 +37,7 @@
             'lxml',
             'mock',
             'plone.app.robotframework>0.9.16',
+            'robotframework-debuglibrary',
```

Comment:
@MrTango I don't think this should be declared as a dependency here. I am sure this is already declared somewhere in plone.app.robotframework. cc @datakurre